core/controllers/editor_test.py

```diff
@@ -25,9 +26,12 @@
 from core.domain import stats_domain
 from core.domain import rights_manager
 from core.domain import rule_domain
+from core.platform import models
 from core.tests import test_utils
 import feconf
 
+(user_models,) = models.Registry.import_models([models.NAMES.user])
```

Comment:
(NB our messages crossed. I think both uses are still fine; the alternative seems pretty clunky in either case.)